about/team/index.md

```diff
@@ -2,61 +2,109 @@
-  complex C++ and design issues. Vadim is based in France and runs his own
-  consultancy company, [TT-Solutions](http://www.tt-solutions.com/).
-
-* [Dr. Stefan Csomor](mailto:csomor@advancedconcepts.ch) - Another medical
-  doctor, Stefan Csomor wrote and maintains the wxMac, the port to OS X. He
-  runs [Advanced Concepts AG](http://www.advanced.ch/), a Swiss software
+* [Dr. Vadim Zeitlin](mailto:vadim@wxwidgets.org) - Vadim is the author of big
+  parts of, and maintainer of, wxMSW port, global coordinator of the wxWidgets
+  project and responsible for making new releases (or guilty of not making
+  them in time, depending on the point of view). Vadim is based in France and
+  runs his own consultancy company, [TT-Solutions](https://www.tt-solutions.com/).
+
+* [Dr. Stefan Csomor](mailto:csomor@advancedconcepts.ch) - An actual medical
+  doctor, Stefan Csomor wrote and maintains wxMac, the port to OS X, and wxiOS.
+  He runs [Advanced Concepts AG](http://www.advanced.ch/), a Swiss software
   consultancy specialising in medical software.
 
+* [Paul Cornett](mailto:wxml-paul@bullseye.com) - Paul is the maintainer of
+  wxGTK.
+
+* [Robin Dunn](mailto:robin@alldunn.com) - Robin works on the very popular
+  [wxPython](http://wxpython.org/) project, which allows using wxWidgets from
+  Python, and also contributed many fixes and improvements to wxWidgets
+  itself.
+
+* [Václav Slavík](mailto:vaclav.slavik@matfyz.cz) - Václav worked on wxHTML, the
+  XRC system, the bakefile-based build system and many other areas. He is also
+  the author of [Poedit](https://poedit.net/), the powerful and intuitive
+  translation editor.
+
+* [Bryan Petty](mailto:bryan@ibaku.net) - Bryan contributed to several areas
+  of wxWidgets and has taken care of hosting wxTrac (now retired), wxWiki and
+  wxWidgets documentation site, without speaking of this website itself, for
+  many years.
+
+
+## Other Active Developers
+
+Thanks to the following people for their continued contributions to wxWidgets:
+
+* Artur Wieczorek who is the maintainer of wxPropGrid library and the resident
+  graphics expert.
+
+* Maarten Bent who is a very regular contributor to wxMSW, implementing big
+  parts of high DPI support in it, maintainer of the CMake-based build system
+  and much more.
+
+* Tobias Taschner who has added several entire components of wxWidgets,
+  including, but not limited to, `wxWebRequest` and the related classes, and
+  `wxWebViewEdge` backend.
+
+* Scott Talbert who added EGL support in `wxGLCanvas` and more.
+
+* Ulrich Telle who added `wxUILocale` class and contributed other fixes.
+  Ulrich is also the author of several popular wxWidgets-related libraries,
+  such as [wxSQLite3](https://github.com/utelle/wxsqlite3),
+  [wxPdfDocument](https://github.com/utelle/wxpdfdoc) and
+  [wxChartDir](https://github.com/utelle/wxchartdir).
+
+* PB has contributed many miscellaneous improvements to wxWidgets and
```

Comment:
Thanks, but I believe I (= PB) belong only in the group listing, since I have not provided even one significant contribution (and I also am the only one not using their full name).

Reply:
> Thanks, but I believe I (= PB) belong only in the group listing,

Frankly, I think otherwise because all the small contributions to the code do add up and I'm also grateful for your comments, testing etc. But let me know if you _really_ don't want to have this entry and I'll remove it.

> (and I also am the only one not using their full name).

Having an anonymous benefactor/contributor makes wxWidgets more mysterious and appealing :-)

Reply:
OK, I understand.